### PR TITLE
[core] Ensure TRG timeouts are obeyed + add polling timeout

### PR DESCRIPTION
**Pull request by teo**

### CODE REVIEW

File: core/integration/trg/plugin.go [hunk 1]
```diff
@@ -853,6 +866,10 @@ func (p *Plugin) CallStack(data interface{}) (stack map[string]interface{}) {
 			return
 		}
 
+		timeout := callable.AcquireTimeout(TRG_START_TIMEOUT, varStack, "RunStart", envId)
+		ctx, cancel := context.WithTimeout(context.Background(), timeout)
+		defer cancel()
+
```
Comment (justonedev1) [inline core/integration/trg/plugin.go:872]: why are you creating the context with timeout here and not next to the place where it is needed? (line 889)

**Review by justonedev1 — COMMENTED** [review]
There are a lot of new timeouts, so I hope that you have verified that adding them will not break anything by timing out sooner than expected.

**Comment by teo** [comment]
> There are a lot of new timeouts, so I hope that you have verified that adding them will not break anything by timing out sooner than expected.

This is my concern too, breaking existing behaviour, but I've been told the timeouts already set in readout-dataflow (which I also used as defaults, so `5s`) are realistic.

File: core/integration/trg/plugin.go (hunk 1 above)
Comment (teo) [reply core/integration/trg/plugin.go:872]: I just did it in the same spot as in `ddsched`. Shouldn't change much either way, as between here and the call we only have declarations and a Kafka event.

**Pull request merged**